control_filters.xml

```diff
@@ -0,0 +1,18 @@
+<class_libraries>
+    <library path="low_pass_filter">
+      <class name="control_filters/LowPassFilterDouble"
+             type="control_filters::LowPassFilter&lt;double&gt;"
+             base_class_type="filters::FilterBase&lt;double&gt;">
```

Comment:
This nomenclature doesn't look nice... It hinders readability very much. We could ignore this field in `xml` checks. But this is also not great.

@bmagyar which "evil" is less?

Reply:
The syntax is also used in the low-level base package https://github.com/ros/filters/blob/ros2/default_plugins.xml

Reply:
@guihomework a good point! Then we should go this way. @bmagyar do you agree?

Reply:
Feel like you are using Internet Explorer in the past :D nah guys it's ok, it's mostly readable